backend/manager/modules/vdsbroker/src/main/java/org/ovirt/engine/core/vdsbroker/builder/vminfo/LibvirtVmXmlBuilder.java

```diff
@@ -2980,6 +2979,9 @@ void writeVideo(VmDevice device) {
         writer.writeStartElement("video");
 
         writer.writeStartElement("model");
+        boolean mdevDisplayOn = MDevTypesUtils
+                .getMdevs(vmInfoBuildUtils.getVmDevices(vm.getId()), VmDeviceType.VGPU).stream()
+                .anyMatch(mdev -> !Boolean.TRUE.equals(mdev.getSpecParams().get(MDevTypesUtils.NODISPLAY)));
```

Comment:
that looks ok but maybe for the time being all devices should be set the same way in regards to no-display

Reply:
This code attempts to be backward compatible in some sense and sets the same nodisplay value for all the vGPU devices when a VM starts. I don't understand your comment exactly — do you mean that nodisplay should be a VM rather than a device property?

Reply:
This has changed and now display is set for each of the devices individually.

Reply:
Actually not, I was looking at writeVGpu instead of writeVideo.

I don't know what would be a good way to resolve this.